python/tvm/_ffi/runtime_ctypes.py

```diff
@@ -217,13 +216,15 @@ class Device(ctypes.Structure):
         "stackvm": 1,
         "cpu": 1,
         "c": 1,
+        "hybrid": 1,
```

Comment:
what is hybrid?

Reply:
Not sure about it. There is no comment or RFC.
https://github.com/apache/tvm/blob/main/src/target/target_kind.cc#L400-L401

Reply:
@masahi there is a legacy thing called TE hybrid script. Probably it’s related?